packages/shared-ui/src/elements/welcome-panel/project-listing.ts

```diff
@@ -447,6 +447,24 @@ export class ProjectListing extends LitElement {
 
     this.addEventListener("click", this.#hideBoardServerOverflowMenuBound);
 
+    for (const boardServer of this.boardServers) {
+      const closuredName = boardServer.name;
+      boardServer.addEventListener(
+        GraphBoardServerRefreshEvent.eventName,
```

Comment:
Let's create BoardServer-native event for this, like "savestatuschange"

https://github.com/breadboard-ai/breadboard/blob/840a88a6b267d207ad9450e8d063e389ba69c45f/packages/breadboard/src/loader/types.ts#L347